lib/bank_account/account.rb

```diff
@@ -0,0 +1,44 @@
+module BankAccount
+  class Account
+  attr_accessor :id, :initial_balance
+
+    def initialize(id, initial_balance)
+      ## optimize raise_error, initialize
+      if initial_balance < 0
+        raise ArgumentError.new("You cannot open an account with a negative balance.")
+      else
+      @id = id.round(2)
+      @initial_balance = initial_balance.round(2)
+      @balance = initial_balance.round(2)
+      end
```

Comment:
Since you're using a `@balance` instance variable you can actually just replace your `initial_balance` attr_accessor with an attr_accessor for `balance` and then you only need the local `initial_balance` var in the initialize.